src/PKSim.Core/Model/Template.cs

```diff
@@ -35,25 +37,87 @@ public static bool Is(this TemplateType templateType, TemplateType typeToCompare
       {
          return (templateType & typeToCompare) != 0;
       }
+
+      public static bool SupportsReference(this TemplateType templateType) =>
```

Comment:
@Yuri05 We want to support references for remote templates. I really want to avoid asking the question for the user if  Idon't have to. So with this, I filter out types not supporting reference. I added Individual and Population for the Expressionprofile that will come very soon